### DISCUSSION TITLE
群晖部署多版本青龙问题

### DISCUSSION THREAD

**Discussion by Dhy827** [whyour/qinglong#1755] [Q&A]
whyour/qinglong:latest会和whyour/qinglong:2.10.13镜像合并显示，在群晖docker里面手动部署最新版的青龙部署完后会默认是2.10.13版本不是最新版本，
![Snipaste_2022-12-27_14-46-21](https://user-images.githubusercontent.com/55142573/209623885-fc6589e5-76bf-42bc-8b9a-ba650a56a58a.jpg)
部署的青龙环境也是2.1013的，就算后期在面板手动升级到最新版也会出现各种各样的问题
![Snipaste_2022-12-27_14-43-14](https://user-images.githubusercontent.com/55142573/209623517-fb71657c-8d8e-44c3-a3ba-58ede0639ebb.jpg)

**Comment by whyour** [comment]
改下群晖的docker源，改成官方的

**Reply by Dhy827** [reply]
![Snipaste_2023-02-06_16-53-09](https://user-images.githubusercontent.com/55142573/216927517-a58df0a3-25e3-4332-9338-23a344db1ba3.jpg)
用的是官方源，用了阿里云加速
